app/services/users/profile.rb

```diff
@@ -162,22 +164,34 @@ def mpi_profile
       else
         error_hash = Users::ExceptionHandler.new(user.mpi_error, 'MVI').serialize_error
         scaffold.errors << error_hash
-        log_external_service_error(error_hash)
+        log_external_service_error(error_hash, 'mpi_profile')
         nil
       end
     end
     # rubocop:enable Metrics/MethodLength
 
     def veteran_status
-      {
+      veteran_status_object = {
         status: RESPONSE_STATUS[:ok],
-        is_veteran: user.veteran?,
-        served_in_military: user.served_in_military?
+        is_veteran: nil,
+        served_in_military: nil
       }
+
+      if user.edipi.blank?
+        Rails.logger.info('Skipping VAProfile veteran status call, No EDIPI present',
+                          user_uuid: user.uuid,
+                          loa: user.loa)
+
+        return veteran_status_object
+      end
+
+      veteran_status_object[:is_veteran] = user.veteran?
+      veteran_status_object[:served_in_military] = user.served_in_military?
+      veteran_status_object
```

Comment:
Here we tackle the external service error from VAProfile (503 RuntimeError invalid edipi). If the user doesn't have an edipi, we do not let the service raise the error anymore.